src/api/java/com/minecolonies/api/util/InventoryUtils.java

```diff
@@ -133,7 +133,7 @@ public static Item getItemFromBlock(final Block block)
     @NotNull
     public static Boolean isItemStackEmpty(@Nullable final ItemStack stack)
     {
-        return stack == null || stack == EMPTY || stack.getCount() <= 0;
+        return stack == null || stack == EMPTY || stack.getItem() != null || stack.getCount() <= 0;
```

Comment:
Two things:

1) I removed this on purpose, because ItemStack.getItem will never be null.
```
public Item getItem()
    {
        return this.isEmpty || this.delegate == null ? Item.getItemFromBlock(Blocks.AIR) : this.delegate.get();
    }
```
If you create an ItemStack with a null item, isEmpty will be true. So this method will return `Item.getItemFromBlock(Blocks.AIR)`.

2) With your change you've made this function always return true. Every ItemStack we will treat as empty